create-customer/src/test/java/no/unit/nva/customer/create/CreateCustomerHandlerTest.java

```diff
@@ -82,13 +82,13 @@ public void requestToHandlerReturnsCustomerCreated() throws Exception {
 
     private Map<String, Object> getRequestHeaders() {
         return Map.of(
-                CONTENT_TYPE, APPLICATION_JSON,
-                ACCEPT, APPLICATION_JSON);
+                CONTENT_TYPE, ContentType.APPLICATION_JSON.getMimeType(),
```

Comment:
This is repeated in GetAllCustomersHandlerTest.java:85 and GetCustomerHandlerTest.java:119
 maybe split into Util.